### PR TITLE
Add tests for molecules select form field

### LINKED ISSUE #26: Backfill tests for Molecules/SelectFormField
Using other component tests as a guide, write automated tests to cover important scenarios in component. Try to avoid granular assertions of the markup. Stay as high-level as possible to avoid test fragility.

### PR DESCRIPTION
**Pull request by SaidShah**
- [x] Related GitHub issue(s) linked in PR description
- [x] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [x] Code builds cleanly (no _additional_ warnings or errors)
- [x] Manually tested
- [x] Automated tests are passing
- [x] No _decreases_ in automated test coverage
- [x] Documentation updated (readme, docs, comments, etc.)
- [x] Localization: No hard-coded error messages in code files (_minimally_ in string constants)
- [x] New component and/ or properties? Make sure to update storybook

Closes #26

### CODE REVIEW

File: src/molecules/form-fields/select-form-field.test.tsx [hunk 1]
```diff
@@ -1,9 +1,171 @@
...
+                id={uuid()}
+                values={[{ value: selectValue, label: selectLabel }]}
+            />
+        );
+
+        // Assert
+        expect(getByText(label)).not.toBeNil();
+    });
+
+    test("when has errorsMessages prop, renders select form field with error message", () => {
```
Comment (wintondeshong) [inline src/molecules/form-fields/select-form-field.test.tsx:31]: @SaidShah as with the other PR, please expand the arrangement when passing arrays to atleast have multiple values. to ensure the system under test handles those values

File: src/molecules/form-fields/select-form-field.test.tsx [hunk 2]
```diff
@@ -1,9 +1,171 @@
...
+import {
+    SelectFormField,
+    InvalidSelectFormValueClass,
+} from "./select-form-field";
+import faker from "faker";
+import uuid from "uuid";
 
 describe("SelectFormField", () => {
-    test.skip("TODO - https://github.com/AndcultureCode/AndcultureCode.JavaScript.React.Components/issues/26", () => {});
+    test("when default props, renders SelectFormField with label", () => {
```
Comment (wintondeshong) [inline src/molecules/form-fields/select-form-field.test.tsx:11]: suggestion: remove 'SelectFormField' from test names as it is inferred and automatically added by outermost test fixture name of "SelectFormField"

File: src/molecules/form-fields/select-form-field.test.tsx [hunk 3]
```diff
@@ -1,9 +1,171 @@
...
+                errorMessages={[testErrorMessage]}
+                values={[{ value: selectValue, label: selectLabel }]}
+            />
+        );
+
+        // Assert
+        expect(getByText(testErrorMessage)).not.toBeNil();
+    });
+
+    test("when has errorsMessage prop, renders select form field with error message", () => {
```
Comment (wintondeshong) [inline src/molecules/form-fields/select-form-field.test.tsx:53]: same as above

**Review by wintondeshong — CHANGES_REQUESTED** [review]
@SaidShah once the test of multiple error messages is expounded upon, we are good to go for approval

File: src/molecules/form-fields/select-form-field.test.tsx (hunk 3 above)
Comment (SaidShah) [reply src/molecules/form-fields/select-form-field.test.tsx]: Thanks for the feedback. I made those changes if you want to take a second look.

**Review by wintondeshong — APPROVED** [review]
@SaidShah great! thanks for the quick turn around on the revisions. approved 👍

**Pull request merged**